projects/ngrx.io/content/guide/router-store/default-router-state-serializer.md

```diff
@@ -0,0 +1,3 @@
+# Default Router State Serializer
```

Comment:
This doesn't need a separate file. I think it should go above `Custom Router State Serializer` in the configuration doc

Reply:
I agree with you @brandonroberts. I have done so based on the following sentence from @timdeschryver on #1642. If you guys, I will change it accordingly.

> This should be a page under [Router Store](https://ngrx.io/guide/router-store)

Reply:
Sorry, my bad 😅